eip_7594/src/lib.rs

```diff
@@ -82,11 +80,10 @@ pub fn verify_kzg_proofs<P: Preset>(data_column_sidecar: &DataColumnSidecar<P>)
         }
     );
 
-    let (row_indices, col_indices): (Vec<_>, Vec<_>) = column
+    let col_indices = column
         .iter()
-        .zip(0_u64..)
-        .map(|(_, row_index)| (row_index, index))
-        .unzip();
+        .map(|_| *index)
+        .collect::<Vec<u64>>();
```

Comment:
this is just a refactor since we don't need row_indices?
`let col_indices: Vec<u64> = vec![index; column.len()];`

Reply:
that make sense, thank you

Reply:
that make sense, thank you